cfe_internal/enterprise/federation/federation.cf

```diff
@@ -530,18 +530,40 @@ bundle agent federation_manage_files
 
 bundle agent postgres_config
 # @brief Customize postgres config for superhub
+# @variable cfengine_enterprise_federation.postgres_config.shared_buffers - Sets the maximum number of locks per transaction.
```

Comment:
Shouldn't `cfengine_enterprise_federation` be a namespace? What is it otherwise?

Reply:
Good eye! Will fix.